### ISSUE TITLE
Json doubles ending in `.0` are parsed weirdly - Chart view

### ISSUE THREAD

**Issue by FMorschel** [thunderclient/thunder-client-support#1518]
**Describe the bug**
I have a response for my data:

<details> 

<summary> Response </summary>

```
{
  "data": [
    {
      "ordem": 48685,
      "valor_total": 702800.0,
    },
    {
      "ordem": 48768,
      "valor_total": 0.0,
    },
    {
      "ordem": 48769,
      "valor_total": 50710.0,
    },
    {
      "ordem": 48774,
      "valor_total": 269.7,
    },
    {
      "ordem": 48781,
      "valor_total": 206.7,
    },
    {
      "ordem": 48782,
      "valor_total": 0.0,
    }
  ]
}
```

</details>

And inside my chart view table, I can't show the second value because it displays as:

![Weird show of double inside html](https://github.com/thunderclient/thunder-client-support/assets/52160996/25d211b9-8e66-41bb-94dc-0d8fbb8574f7)

I tested by showing the full data and the following appeared:

<details>

<summary> <code> chart_data </code> </summary> 

```
[
  {
    "ordem": 48685,
    "valor_total": {
      "numberStr": "702800.0"
    }
  },
  {
    "ordem": 48768,
    "valor_total": {
      "numberStr": "0.0"
    }
  },
  {
    "ordem": 48769,
    "valor_total": {
      "numberStr": "50710.0"
    }
  },
  {
    "ordem": 48774,
    "valor_total": 269.7
  },
  {
    "ordem": 48781,
    "valor_total": 206.7
  },
  {
    "ordem": 48782,
    "valor_total": {
      "numberStr": "0.0"
    }
  }
]
```

</details>

**To Reproduce**
- Create an API get that returns double values with `.0` ending.

- Get the value inside the chart HTML by calling  `tc.chartHTML(html, responseInnerJson);`

**Expected behavior**
I expect double values to be parsed right (really weird because integers parse with no problems as well and js has no differentiation)

**Platform:** 
 - OS: Windows_NT x64 10.0.22621
 - vscode version: 1.88.0
 - node version: 18.18.2
 - extension version: 2.20.3

**Are you using the free version/paid version/trial:**
Free version.

**Comment by rangav** [comment]
Hi @FMorschel, thanks for reporting, Will fix it ASAP.

**Comment by rangav** [comment]
Alternative you can use this code

```js
var response = JSON.parse(tc.response.text);
var html = `html code`;

tc.chartHTML(html, response.data);
```

**Comment by FMorschel** [comment]
Tested it and it worked, thanks for the workaround.

**Comment by rangav** [comment]
This bug is fixed, Please update extension to `v2.21.4`
